pycheribuild/projects/run_fvp.py

```diff
@@ -198,10 +198,8 @@ def execute_fvp(self, args: list, disk_image_path: Path = None, firmware_path: P
                 # and no way to recover otherwise anyway given we can't change
                 # port forwarding after running.
                 pre_cmd += ["-p", str(docker_host_ap_port) + ":" + str(default_ap_port)]
-            if ssh_port is not None:
-                pre_cmd += ["-p", str(ssh_port) + ":" + str(ssh_port)]
-            if gdb_port is not None:
-                pre_cmd += ["-p", str(gdb_port) + ":" + str(gdb_port)]
+            for p in tcp_ports:
```

Comment:
Will this crash when tcp_ports is None?

Reply:
Oh, probably.  I guess I should make the default `[]`.

Reply:
Using {} or [] as default arguments is generally not a good idea in Python since you end up mutating the default for the following calls if you do make any changes...

Reply:
Right, yes; thanks for the reminder.